apps/endatix-hub/app/(main)/forms/[formId]/ui/form-editor.tsx

```diff
@@ -255,7 +269,12 @@ function FormEditor({ formJson, formId, formName, options }: FormEditorProps) {
             </span>
           )}
         </div>
-        <div className="flex items-center gap-2" >
+        <div className="flex items-center gap-2">
+          {hasUnsavedChanges && (
+            <span className="text-yellow-600 text-xs border border-yellow-600 px-2 py-0.5 rounded-full whitespace-nowrap">
```

Comment:
Yellow on a white background is not the most contrastful combination. I would go for green and make the text bold, or maybe even try to make the background green anf the text white.

Reply:
You're right, thanks for the suggestion. I'll make changes but I only think is green the right for this? We want to warn the user and green doesn't sound to me really for warning. What do you think?

Reply:
Yellow doesn't work on white, unless you make the whole background yellow and make the text black. Red is wrong, because it suggests an error. Green is not ideal for the reasons you stated, but it grabs attention better, because everything else we have in color is in blue.
Most sites we looked at for reference don't actually have any color for this status label. It is black and white. I believe that's the case with Github too.